app/views/secure_messaging.py

```diff
@@ -50,24 +66,10 @@ def reply_message():
 
     if request.method == 'POST':
         logger.info("Reply to Message")
-        data = {'msg_to': 'BRES', 'msg_from': 'tom@gmail.com', 'subject': 'reply_subject', 'body': request.form['secure-message-body'],
+        data = {'urn_to': 'BRES', 'urn_from': 'tom@gmail.com', 'subject': 'reply_subject', 'body': request.form['secure-message-body'],
                 'thread_id': 'test', 'collection_case': 'test', 'reporting_unit': 'test', 'survey': 'BRES'}
-        response = requests.post(SecureMessaging.CREATE_MESSAGE_API_URL, data=json.dumps(data), headers=headers)
+
```

Comment:
response = requests.post(SecureMessaging.CREATE_MESSAGE_API_URL, data=json.dumps(data), headers=headers) is the correct line